dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -268,7 +270,10 @@ case class BinaryOutcomeDLCWithSelf(
     */
   def executeDLC(
       oracleSigF: Future[SchnorrDigitalSignature],
-      local: Boolean): Future[(Transaction, BitcoinUTXOSpendingInfo)] = {
+      local: Boolean,
+      rpcClientAndAddressOpt: Option[
```

Comment:
there should not be a dependency on bitcoind rpc client inside of the DLC project source code. This breaks bitcoin-s modularity for DLCs. Signing/construction/storing of DLC contracts should not depend on a bitcoind rpc client.